cmd/loadgen-scaler/scaler.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/prometheus/prombench/pkg/provider/k8s"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+type scale struct {
+	k8sClient         *k8s.K8sClient
+	scaleUpReplicas   int32
+	scaleDownReplicas int32
+	intervalMinutes   int
+}
+
+func main() {
+
+	app := kingpin.New(filepath.Base(os.Args[0]), "The Prombench-Scaler tool")
+	app.HelpFlag.Short('h')
+
+	k, err := k8s.NewK8sClient()
+	if err != nil {
+		fmt.Fprintln(os.Stderr, errors.Wrapf(err, "Error creating k8s client inside the k8s cluster"))
+		os.Exit(2)
+	}
+
+	s := scale{k8sClient: k}
```

Comment:
best to have an initialisation like `scaler.New()`. Can be in the same file just a bit more single purpose and keeping the main func as short as possible.